S21/rkarmuri/ConvertToLowerCase.java

```diff
@@ -0,0 +1,98 @@
+//Method to convert given string to lowercase
+package com.company;
+
+import java.util.Locale;//Package to accept multiple languages
+
+public class ConvertToLowerCase {
+    public String convert(String wordSample, String language) {
+        System.out.println("Input word: " + wordSample);
+        System.out.println("Input language tag: " + language);
+        String wordLowerCase = wordSample.toLowerCase();//Conversion in English
+        String trazString = wordSample.toLowerCase(Locale.forLanguageTag("az"));//Conversion in Turkish and Azerbaijani
+        String russString = wordSample.toLowerCase(Locale.forLanguageTag("ru-Latn"));//Conversion in Russian
+        String hungarianString = wordSample.toLowerCase(Locale.forLanguageTag("hu"));//Conversion in Hungarian
+        //Conversion cases for Greek
+        String greekString = wordLowerCase.replace('Σ', 'σ').replace('Ε', 'ε').
+                replace('Α', 'α').replace('Β', 'β').
+                replace('Γ', 'γ').replace('Π', 'π').
+                replace('Ό', 'ό').replace('Λ', 'λ').
+                replace('Η', 'η').replace('Δ', 'δ').
+                replace('Ζ', 'ζ').replace('Θ', 'θ').
+                replace('Μ', 'μ').replace('Ν', 'ν').
+                replace('Ξ', 'ξ').replace('Ρ', 'ρ').
+                replace('Ω', 'ω').replace('Ψ', 'ψ').
+                replace('Χ', 'χ').replace('Φ', 'φ').
+                replace('Υ', 'υ').replace('Τ', 'τ');
```

Comment:
Do you think we can combine the letters and their replacement into arrays or list, so it would be more readable?

Reply:
It is a good approach to put all the letters in an array, but to read and run the arrays list makes it hard for me.